cfme/common/__init__.py

```diff
@@ -81,7 +81,12 @@ def _assign_unassign_policy_profiles(self, assign, *policy_profile_names):
                 view.policy_profiles.check_node(policy_profile)
             else:
                 view.policy_profiles.uncheck_node(policy_profile)
-        view.save.click()
+        # Save button will be disabled in case of no changes done to policy,
+        # In this case click on Cancel
+        if view.save.disabled:
+            view.cancel.click()
+        else:
+            view.save.click()
```

Comment:
**REQUIRED**
It shouldn't be done like that. `ManagePoliciesFromDetails` should have `fill()` method that returns a boolean value. True if new values differs, otherwise False. And then you can choose the correct button to click.
```python
changed = view.fill(policy_profile_names, assign)
if changed:
    view.save.click()
else:
    view.cancel.click()
```

Reply:
Add new method to ManagePoliciesView, check_policy_assign_unassign.
I call this method before performing assign\Uassign to policy,
According to that i chose to perform changes + Save or Cancel.